scripts/wsrep_sst_xtrabackup-v2.sh

```diff
@@ -1705,6 +1710,9 @@ function initialize_pxb_commands()
                 \$encrypt_backup_options --stream=\$sfmt \
                 --xtrabackup-plugin-dir="$pxb_plugin_dir" \
                 --target-dir=\$itmpdir 2> >(logger -p daemon.err -t ${ssystag}innobackupex-backup)"
+            INNODECOMPRESS="${pxb_bin_path} $disver --parallel=${decompress_threads} --decompress --remove-original \
```

Comment:
if decompress fails, do we retry the operation on the same directory?

Reply:
No, if decompress fails, SST script fails and node fails to join.
Then it is restarted, SST starts again, old directory is removed, then downloaded, and so on

Reply:
Thanks. OK to add --remove-original :)

Reply:
do you have any PXC test with the logs? showing SST joiner able to decompress zstd backups from donor

Reply:
Good point. I have enabled compression in galera_sst_xtrabackup-v2 MTR test.